demos/src/scss/resized.scss

```diff
@@ -9,5 +9,5 @@ $o-grid-gutters: (default: 5px);
 @import 'common';
 
 .test body:after {
-	content: '$o-grid-layouts: S: 400px, M: 800px, L: 1000px, $o-grid-gutters: #{$o-grid-gutters}';
+	content: '$o-grid-layouts: S: 400px, M: 800px, L: 1000px, $o-grid-gutters: 5px';
```

Comment:
Possibly. Although as this is just for the demo (and also not a bit of code that's likely to be "copy-pasted"), I don't think there's a huge issue in it being hardcoded.